### PR TITLE
post: Tina GraphQL API

### PR DESCRIPTION
**Pull request by jeffsee55**
Closes https://github.com/tinacms/marketing-posts/issues/3

### CODE REVIEW

**Comment by jeffsee55** [comment]
I'm going to pull back on this example and instead use the example from the Next.js starter. Making this a little bit conceptual and a little bit more of a demo of how it looks in practice

File: content/blog/using-graphql-with-the-filesystem.md [hunk 1]
```diff
@@ -15,15 +15,32 @@ author: Jeff See
 
 ---
 
+---
+title: Limitations of using the filesystem for web content and how Tina's GraphQL API solves this
```
Comment (DirtyF) [inline content/blog/using-graphql-with-the-filesystem.md:19]: Tina's GraphQL API brings data integrity and solid relationships to Git-based content management

File: content/blog/using-graphql-with-the-filesystem.md [hunk 2]
```diff
@@ -15,15 +15,32 @@ author: Jeff See
...
+date: '2021-04-22T10:00:00.000Z'
+draft: true
+author: Jeff See
+---
+
+- The power of Git
+- The limitations of the Git-based CMS
+  - Relationships: it's complicated.
+- The Power of GraphQL
+- Mixing best of both worlds
```
Comment (DirtyF) [inline content/blog/using-graphql-with-the-filesystem.md:29]: Getting the benefits of GIt _and_ GraphQL

**Comment by jeffsee55** [comment]
Kind of got pulled away late this afternoon so revisiting it this evening. Not quite finished but any feed back is welcome. I hope to have the conclusion wrapped up tomorrow morning.

File: content/blog/using-graphql-with-the-filesystem.md [hunk 3]
```diff
@@ -391,4 +385,4 @@ query BlogPostQuery($relativePath: String!) {
 
 ## Conclusion
 
-TODO
+TODO
```
Comment (DirtyF) [inline content/blog/using-graphql-with-the-filesystem.md:388]: - Sum up the benefits of having type checking on top the filesystem
- Added benefits: better developer experience in Visual Code?
-  invite people to play with the starter and give us feedback

**Review by DirtyF — APPROVED** [review]
LGTM!

**Pull request merged**